Example/package.json

```diff
@@ -33,6 +33,7 @@
     "@babel/polyfill": "^7.2.5",
     "@babel/preset-env": "^7.3.1",
     "@babel/runtime": "^7.4.4",
+    "@react-native-community/cli": "^1.9.4",
```

Comment:
Hi, i found if use this package, `pod install` will give an error `Invalid `Podfile` file: A JSON text must at least contain two octets!.` Because there is no config in @react-native-community/cli-1.9.4 ! Please check!